tools/accuracy_checker/accuracy_checker/metrics/mpjpe_multiperson.py

```diff
@@ -0,0 +1,110 @@
+"""
+Copyright (c) 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import numpy as np
+from ..representation import PoseEstimation3dPrediction, PoseEstimation3dAnnotation
+from .metric import PerImageEvaluationMetric
+
+
+class MpjpeMultiperson(PerImageEvaluationMetric):
+    __provider__ = 'mpjpe_multiperson'
+    annotation_types = (PoseEstimation3dAnnotation,)
+    prediction_types = (PoseEstimation3dPrediction,)
+
+    def __init__(self, config, dataset, name=None, state=None):
+        super().__init__(config, dataset, name, state)
+        self.per_image_mpjpe = []
+        self.meta.update({
+            'scale': 10,
+            'postfix': 'mm'
+        })
+
+    def update(self, annotation, prediction):
+        # since pelvis does not detected, remove it before evaluation
+        kpt_mask = np.ones_like(annotation.x_3d_values[0], dtype=bool)
+        kpt_mask[2] = False
+        annotation.x_3d_values = annotation.x_3d_values[:, kpt_mask]
+        annotation.y_3d_values = annotation.y_3d_values[:, kpt_mask]
+        annotation.z_3d_values = annotation.z_3d_values[:, kpt_mask]
+        if prediction.size:
+            prediction.x_3d_values = prediction.x_3d_values[:, kpt_mask]
+            prediction.y_3d_values = prediction.y_3d_values[:, kpt_mask]
+            prediction.z_3d_values = prediction.z_3d_values[:, kpt_mask]
+
+        matching_result = {
```

Comment:
Possibly, you can use just 2 lists - `gt_coordinates` and `predicted_coordinates` without dictionary.